news/2023/2023-06-20-beach-episode-art-contest.md

```diff
@@ -14,26 +14,26 @@ It's been hot as hell this year, so what better way to welcome things in than wi
 
 **Draw any of the official or community mascots having fun at a beach!**
 
-That's it! If it's an activity someone can, might, or reasonably has at a beach before, it's fair game if you draw any one of our mascots doing them.
+That's it! If it's an activity someone can, might, or reasonably has had at a beach before, it's fair game if you draw any one of our mascots doing them.
 
 In case you live in a cave and have absolutely no idea what a [beach episode](https://tvtropes.org/pmwiki/pmwiki.php/Main/BeachEpisode) is even referring to, here's some ideas to get your creative cogs churning:
 
-- Mascots engaged in high-stakes hitcircle beach volleyball
+- Mascots engaged in high-stakes hit circle beach volleyball
 - Yuzu and the Taikonator engaged in sand castle siege warfare
 - Mari decimating a watermelon with a stick alarmingly close to Mani's head (who is of course, buried in the sand)
 - Mocha hunting for crabs in the surf
 - pippi surfing a bitchin' shorey with a pair of shades on (no we will not elaborate on this slang)
 - Seaside cooking stall shenanigans with a shop owner suspiciously similar to the Dean "peppy" Herbert
 - Cinematic scenes of playing with fireworks on the beach at sunset (do not forget the bucket)
-- ...and basically anything else you can think of that might feature in a beach episode!
+- ...and basically anything else you can think of that might be featured in a beach episode!
```

Comment:
what

Reply:
because "feature" is a transitive verb. the "thing you can think of" is being "featured", and the "beach episode" "features" the thing